src/landing/LandingPage.js

```diff
@@ -12,38 +12,33 @@ const LandingPage = () => {
   const [form, setForm] = useState('Login');
   const [usernameState, setUsernameState] = useState(null);
   const [passwordState, setPasswordState] = useState(null);
+  const [loginResult, setLoginResult] = useState(null);
+  const [registerResult, setRegisterResult] = useState(null);
 
   const [getUsers, {
-    data: queryData, error: queryError, loading: queryLoading,
+    data: queryData, error: queryError,
   }] = useLazyQuery(GET_USERS);
 
   useEffect((() => {
     if (queryData && usernameState && passwordState) {
-      console.log(queryData);
       if (queryData.users.find((user) => user.username === usernameState
           && user.password === passwordState)) {
-        alert(`Login successful for ${usernameState}`);
+        setLoginResult('success');
```

Comment:
You have some strings here which you use in another file. You can extract them in an object in a separate file:

```js
// constants.js
const loginStates = {
    success: 'success',
    credentialsError: 'credentialsError',
    serverError: 'serverError'
} 
```

Then you can import them here and in the other file where they are needed. This is more future-proof as it avoids things getting out of sync and it also keeps things DRY.

Reply:
Updated